Move PR template and add manual testing checkbox (#446)
cagov/ui-claim-tracker: .github/pull_request_template.md

- [ ] Relevant documentation (e.g. READMEs, Technical Foundation) updated
- [ ] Issue AC are copied to this PR & are met

> - [ ] Manual Browser Testing performed (via local browser or BrowserStack w/ modheader)

Reviewer: Moving modifier to beginning (I think this is what you mean)

```suggestion
- [ ] Manual Browser Testing performed (w/ modheader, locally or in BrowserStack)
```

Author: ```suggestion
- [ ] Manual Browser Testing performed (with modheader, either locally or on BrowserStack)
```
